.circleci/config.yml

```diff
@@ -8,7 +8,8 @@ jobs:
     docker:
       - image: ubuntu:18.04
     steps:
-      - run: apt-get update ; apt-get install -y make git curl
+      - run: apt-get update
+      - run: apt-get install -y --no-install-recommends make git curl
```

Comment:
According to [Docker-build documentation](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run) it is not recommended to have _update_ and _install_ in separate steps. Not sure under what circumstances that is a problem though.

Reply:
I fixed that with my latest commit.